Dockerfile

```diff
@@ -1,135 +1,10 @@
-	rm /opt/fastqc_v0.11.5.zip && \
-	cd /opt/FastQC && \
-	shopt -s extglob && \
-	rm -rfv !\("fastqc"\|*.jar\) && \
-	chmod 777 * && \
-	ln -s /opt/FastQC/fastqc /usr/local/bin/
-# Install Pandoc (required by reporter) RUN aria2c https://github.com/jgm/pandoc/releases/download/1.19.2.1/pandoc-1.19.2.1-1-amd64.deb -q -o /opt/pandoc-1.19.2.1-1-amd64.deb && \
-	dpkg -i /opt/pandoc-1.19.2.1-1-amd64.deb && \
-	rm /opt/pandoc-1.19.2.1-1-amd64.deb
-# Install PyPy RUN aria2c https://bitbucket.org/squeaky/portable-pypy/downloads/pypy-5.9-linux_x86_64-portable.tar.bz2 -q -o /opt/pypy-5.9-linux_x86_64-portable.tar.bz2 && \
-	tar xf /opt/pypy-5.9-linux_x86_64-portable.tar.bz2 --use-compress-prog=pbzip2 -C /opt/ && \
-	rm /opt/pypy-5.9-linux_x86_64-portable/README.rst /opt/pypy-5.9-linux_x86_64-portable.tar.bz2 && \
-	ln -s /opt/pypy-5.9-linux_x86_64-portable/bin/pypy /usr/local/bin/
-# Install BEDOPS RUN aria2c https://github.com/bedops/bedops/releases/download/v2.4.29/bedops_linux_x86_64-v2.4.29.tar.bz2 -q -o /opt/bedops_linux_x86_64-v2.4.29.tar.bz2 && \
-	tar xf /opt/bedops_linux_x86_64-v2.4.29.tar.bz2 --use-compress-prog=pbzip2 -C /opt/ && \
-	ln -s /opt/bin/* /usr/local/bin/ && \
-	rm /opt/bedops_linux_x86_64-v2.4.29.tar.bz2
-# Install AfterQC # Use PyPy to run AfterQC as default RUN aria2c https://github.com/OpenGene/AfterQC/archive/v0.9.7.tar.gz -q -o /opt/AfterQC-0.9.7.tar.gz && \
-	tar xf /opt/AfterQC-0.9.7.tar.gz --use-compress-prog=pigz -C /opt/ && \
-	cd /opt/AfterQC-0.9.7 && \
-	make && \
-	perl -i -lape's/python/pypy/ if $. == 1' after.py && \
-	rm -rf Dockerfile Makefile README.md testdata report_sample && \
-	rm editdistance/*.cpp editdistance/*.h && \
-	ln -s /opt/AfterQC-0.9.7/*.py /usr/local/bin/ && \
-	rm /opt/AfterQC-0.9.7.tar.gz
-# Install R package LncPipeReporter（via GitHub) RUN Rscript -e "source('http://bioconductor.org/biocLite.R'); install.packages(c('curl', 'httr')); install.packages('devtools'); devtools::install_github('bioinformatist/LncPipeReporter')"
-# Install GffCompare RUN aria2c https://github.com/gpertea/gffcompare/archive/master.zip -q -o /opt/gffcompare-master.zip && \
-	aria2c https://github.com/gpertea/gclib/archive/master.zip -q -o /opt/gclib-master.zip && \
-	unzip -qq /opt/gffcompare-master.zip -d /opt/ && \
-	unzip -qq /opt/gclib-master.zip -d /opt/ && \
-	rm /opt/gffcompare-master.zip /opt/gclib-master.zip && \
-	mv /opt/gclib-master /opt/gclib && \
-	cd /opt/gffcompare-master && \
-	make release && \
-	rm Makefile README.md gtf_tracking.h *.o *.cpp *.sh && \
-	ln -s /opt/gffcompare-master/gffcompare /usr/local/bin/
-# Install sambamba RUN aria2c https://github.com/biod/sambamba/releases/download/v0.6.7/sambamba_v0.6.7_linux.tar.bz2 -q -o /opt/sambamba_v0.6.7_linux.tar.bz2 && \
-	tar xf /opt/sambamba_v0.6.7_linux.tar.bz2 --use-compress-prog=pbzip2 -C /opt/ && \
-	ln -s /opt/sambamba /usr/local/bin/ && \
-	rm /opt/sambamba_v0.6.7_linux.tar.bz2
+
+FROM nfcore/base
+MAINTAINER Qi Zhao <zhaoqi@sysucc.org.cn>
```

Comment:
MAINTAINER is deprecated, you should use LABEL for that instead: cf [Docker docs](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated)